README.md

```diff
@@ -74,7 +94,7 @@ You can also check the docker container setup scripts for a reference how to bui
 <ramses-sdk>/scripts/docker/runtime-files/build-ramses.sh   -> contains CMake command for building
 ```
 
-3 Copyright / License / Open Source / Third Party
+4 Copyright / License / Open Source / Third Party
 -----------------------------------
 RAMSES original code is copyright BMW Car IT
```

Comment:
I'm not entirely sure that the last line helps anything.  A statement like that in the README tends to stay there even after others have contributed and that's just misleading.  Even with the word "original code", how will people know what the "original code" is....  More typical is to use (C) statements at the top of each source file.  People who contribute patches can add their own (C) line as part of a patch (but often they miss that, and the git commit history keeps the information).  It's still clearer to do that since it is attached to the code itself.

If you really need this statement maybe define the commit of the "original code"?  Alternatively, if the intention is more to say that BMW wrote and contributed the original code and get credit for that, then maybe write it that way?

Reply:
Agree, but not related to this pull request, but to the previous state of README, right?

Reply:
Yes, sure, the comment is on the line that was already there before.   Just like Jeremiah's comment it was a proposal for more README improvement which I thought you might want to include more improvements before merging.